app/src/pages/inside/logsPage/logsGrid/logMessageBlock/logMessageBlock.jsx

```diff
@@ -37,14 +38,16 @@ export class LogMessageBlock extends Component {
 
   render() {
     const { value, refFunction, customProps } = this.props;
+    const markdownModeRegExp = new RegExp(MARKDOWN_MODE, 'g');
```

Comment:
I think it would be enough to seek for this substring only at the beginning of the string (as it was before).